### PR TITLE
Documentation: Added example to DateTime opBinary and reference to core.time.Duration

### PR DESCRIPTION
**Pull request by bachmeil**
There is currently no usage example and no way for the reader to know how to specify a duration.

### CODE REVIEW

**Comment by bachmeil** [comment]
The example is taken from [this forum post](http://forum.dlang.org/post/n3g334$19ga$1@digitalmars.com).

File: std/datetime.d [hunk 1]
```diff
@@ -16153,6 +16153,21 @@ public:
         Params:
             duration = The duration to add to or subtract from this
                        $(LREF DateTime).
+
+        See_Also:
+            $(CXREF time, Duration).
```
Comment (schveiguy) [inline std/datetime.d]: Suggest instead to change Param specification to:

The $(CXREF time, Duration) to add or subtract...

File: std/datetime.d [hunk 2]
```diff
@@ -16153,6 +16153,21 @@ public:
...
+        --------------------
+        import std.datetime, std.stdio;
+        void main()
+        {
+            DateTime oldYear = DateTime(2015, 12, 31, 23, 59, 59);
+            DateTime newYear = oldYear + 1.seconds;
+            Duration diff = newYear - oldYear;
+            writeln(diff); /* 1 sec */
+        }
+        --------------------
```
Comment (schveiguy) [inline std/datetime.d]: You should make this a commented unit test, then it is tested to make sure it remains valid.

File: std/datetime.d (hunk 1 above)
Comment (bachmeil) [inline std/datetime.d]: Yes, that is better. I'm not a Git expert. Is there a way I can edit this pull request, or do I have to close it and create a new one?

**Comment by schveiguy** [comment]
There's also SysTime.opBinary

File: std/datetime.d (hunk 1 above)
Comment (schveiguy) [inline std/datetime.d]: Yes, just push more changes to your branch, the PR will update.

**Comment by bachmeil** [comment]
I've made these changes. Hopefully the SysTime.opBinary example makes sense, as I haven't used that before.

File: std/datetime.d [hunk 3]
```diff
@@ -16171,6 +16179,14 @@ public:
         mixin(format(`return retval._addSeconds(convert!("hnsecs", "seconds")(%shnsecs));`, op));
     }
 
+    ///
+    unittest
+    {
+        DateTime oldYear = DateTime(2015, 12, 31, 23, 59, 59);
+        DateTime newYear = oldYear + 1.seconds;
+        assert(newYear == DateTime(2016, 1, 1, 0, 0, 0));
```
Comment (schveiguy) [inline std/datetime.d]: I know I'm being OCD, but the examples should be consistent -- one uses "auto", the other doesn't, one calls the variables "st1" and "st2", the other calls them "oldYear" and "newYear"

Other than that, LGTM.

**Comment by schveiguy** [comment]
LGTM

**Comment by schveiguy** [comment]
BTW, when you update something, a notification is NOT sent out to reviewers unless you make a comment. I happened to notice your change this time.

**Comment by bachmeil** [comment]
Okay. I assumed you would get a notification.

**Comment by Hackerpilot** [comment]
Auto-merge toggled on

**Pull request merged**